base/fastmath.jl

```diff
@@ -124,6 +125,38 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+@fastmath expr
```

Comment:
oh noes! Jk i can't build on this computer; i don't have all the dependencies installed. I'll push this tomorrow. Thanks! :)